pkg/harvester/models/harvesterhci.io.virtualmachineimage.js

```diff
@@ -48,7 +48,7 @@ export default class HciVmImage extends HarvesterResource {
       {
         action:   'createFromImage',
         enabled:  canCreateVM,
-        icon:     'icon icon-fw icon-spinner',
+        icon:     'icon icon-circle-plus',
```

Comment:
We don't have virtual machine icon in [rancher icon](https://rancher.github.io/icons/) ☹️
Choose another icon instead.

<img width="488" alt="Screenshot 2024-09-10 at 1 36 14 PM" src="https://github.com/user-attachments/assets/217f4735-ab7c-4b76-a99c-c2ba41a6e93d">